src/ctapipe/instrument/optics.py

```diff
@@ -263,3 +270,158 @@ def __repr__(self):
 
     def __str__(self):
         return self.name
+
+
+class PSFModel(TelescopeComponent):
+    """
+    Base component to describe image distortion due to the optics of the different cameras.
+    """
+
+    @abstractmethod
+    def pdf(self, r, f, r0, f0, *args):
+        """
+        Calculates the value of the psf at a given location
+
+        Parameters
+        ----------
+        r : float
+            distance to the center of the camera in meters, location at where the PSF is evaluated
+        f : float
+            polar angle in radians, location at where the PSF is evaluated
+        r0 : float
+            distance to the center of the camera in meters, location from where the PSF is evaluated
+        f0 : float
+            polar angle in radians, location from where the PSF is evaluated
+        Returns
+        ----------
+        psf : float
+            value of the PSF at the specified location
+        """
+        pass
+
+
+class ComaModel(PSFModel):
+    r"""
+    PSF model, describing pure coma aberrations PSF effect
+    asymmetry_params describe the dependency of the psf on the distance to the center of the camera
+    Used to calculate a pdf asymmetry parameter K of the asymmetric radial laplacian of the psf as
+    a function of the distance r to the optical axis
+
+    .. math:: K(r) = 1 - asym_0 \tanh(asym_1 r) - asym_2 r
+
+    radial_scale_params describes the dependency of the radial scale on the distance to the center of the camera
+    Used to calculate width Sr of the asymmetric radial laplacian in the PSF as a of function the distance r to the optical axis
+
+    .. math:: S_{R}(r) & = b_1 - b_2\,r + b_3\,r^2 + b_4\,r^3
```

Comment:
Since you changed the implementation to `np.polyval`, you should update also the definition here:
```
.. math:: S_{R}(r) & = b_1 + b_2\,r + b_3\,r^2 + b_4\,r^3
```